.github/workflows/pr-command.yml

```diff
@@ -82,4 +90,5 @@ jobs:
           ```
           $ CRYOSTAT_IMAGE=${{ steps.push-to-ghcr.outputs.registry-path }} sh smoketest.sh
           ```
-      if: github.repository_owner == 'cryostatio'
+          ${{ github.workflow   }}: ${{steps.workflow-status.outputs.status}}
```

Comment:
As said above with Max, I am not sure we need to print these out. The action table at the bottom of the PR will show all these.

Reply:
However, one idea is that you can group both the amd64 and arm64 image tag into a single PR comment instead of 2 separate ones.